build.gradle

```diff
@@ -44,6 +44,7 @@ dependencies {
     implementation 'com.squareup.okhttp3:okhttp:4.9.1'
     implementation 'com.subshell.simpleframework:simple-xml:2.9.0'
     implementation 'org.slf4j:slf4j-api:1.7.32'
+    implementation 'org.apache.commons:commons-lang3:3.12.0'
```

Comment:
Please avoid adding dependencies. If necessary implement the necessary methods here.

Additional dependencies increase the attack surface in case they contain vulnerabilities. We are responsible to fix/upgrade when we use them.